xml/obs_best_practice_kiwi_editor.xml

```diff
@@ -9,45 +9,165 @@
  xmlns:xlink="http://www.w3.org/1999/xlink" >
  <title>&kiwi; Editor</title>
  <para> You can edit the &kiwi; file associated to your project. It is only
-  possible, at the moment, to edit the repository list. If you are running your
-  own instance of OBS be sure you have the kiwi_image_editor feature enabled in
-  your config/feature.yml file. </para>
+   possible, at the moment, to edit the repository list and packages with type image.
+   If you are running your own instance of OBS be sure you have the kiwi_image_editor feature
+   enabled in your config/feature.yml file. </para>
  <sect2>
   <title>Accessing the &kiwi; Editor</title>
   <para> Go to your package, and upload a file with the <filename
     class="extension">.kiwi</filename> extension (for example,
-   <filename>test.kiwi</filename>), with a valid &kiwi; content. </para>
+    <filename>test.kiwi</filename>), with a valid &kiwi; content.
+  </para>
   <figure>
    <title>Example of a Package with a &kiwi; XML File</title>
    <mediaobject>
     <imageobject>
-     <imagedata fileref="edit_kiwi_package_overview.png"/>
+     <imagedata fileref="kiwi_package_overview.png"/>
     </imageobject>
    </mediaobject>
   </figure>
   <note>
    <para> You should see now a "Edit KIWI" link (next to "Delete package"
     link). </para>
   </note>
-  <para> Click on the "Edit KIWI" link. </para>
+  <para> Click on the "Edit KIWI" link and you will be redirected to the Editor.</para>
   <figure>
    <title>&kiwi; Editor. Show screen</title>
    <mediaobject>
     <imageobject>
-     <imagedata fileref="edit_kiwi_show.png"/>
+     <imagedata fileref="kiwi_editor_show.png"/>
     </imageobject>
    </mediaobject>
   </figure>
-  <para> Click on the "Edit" button. </para>
+  <para> We can see two different sections:
+    <itemizedlist>
+     <listitem>
+      <para>
+        <emphasis>Repositories:</emphasis> Display the repositories set in the Kiwi file.
+      </para>
+      </listitem>
+     <listitem>
+      <para>
+        <emphasis>Packages:</emphasis> Display the packages of the package group with type <emphasis>image</emphasis>.
+      </para>
+     </listitem>
+    </itemizedlist>
+  </para>
+  </sect2>
+  <sect2>
+  <title>Adding Repositories in the &kiwi; Editor</title>
+  <para> To add a new repository you only need click to <emphasis>Add repository</emphasis> link and fill the dialog.
```

Comment:
"you only need" is unecessary. I would just say:
"To add a new repository **click** <emphasis>Add repository</emphasis> link and fill **in** the dialog."